### ISSUE TITLE
Kotlin stdlib has greater resolution priority than jars added via @file:DependsOn annotation

### ISSUE THREAD

**Issue by nikitinas** [Kotlin/kotlin-jupyter#25]
In `krangl` library there is an infix extension function:
```infix fun String.to(that: TableExpression) = ColumnFormula(this, that)```

But compilation of the following code fails:
```
%use krangl
"a" to { it["a"] }
```

Currently the only way to use correct `to` function is to add `import` statement in every cell where it is called:

```
import krangl.*
"a" to { it["a"] }
```

**Comment by nikitinas** [comment]
Youtrack issue: https://youtrack.jetbrains.com/issue/KT-35651

**Comment by ileasile** [comment]
Fixed in 0.8.1.72
